### PR TITLE
Add sitemap and PWA support in landing page

### PR DESCRIPTION
**Pull request by dhararaghu777**
## What is new in this PR

> Added PWA support using the next-pwa library
> Added sitemap files using next-sitemap for better SEO

### CODE REVIEW

File: public/sitemap-0.xml
Comment (anuragxxd) [inline public/sitemap-0.xml:1]: This shouldn't be commited! Right?

File: public/sitemap.xml
Comment (anuragxxd) [inline public/sitemap.xml:1]: This too!

File: public/sw.js
Comment (anuragxxd) [inline public/sw.js:1]: Same!

**Review by anuragxxd — CHANGES_REQUESTED** [review]
Some changes otherwise lgtm

File: public/sitemap-0.xml
Comment (dhararaghu777) [reply public/sitemap-0.xml:1]: Yeah these will be generated during the build, I will remove them and push again

Comment (dhararaghu777) [reply public/sitemap-0.xml:1]: Resolved

File: public/sw.js
Comment (dhararaghu777) [reply public/sw.js:1]: Resolved

**Review by anuragxxd — APPROVED** [review]

**Pull request merged**